handler/population_types.go

```diff
@@ -96,11 +96,77 @@ func (h *PopulationTypes) Get(w http.ResponseWriter, req *http.Request) {
 	h.respond.JSON(ctx, w, http.StatusOK, resp)
 }
 
-// GetAreaTypes is the handler for GET /population-type/{id}/area-types
-// Gets area types from cantabular. If not authenticated, then only return
-// area-types if datasets using it published. Whether published determined by making
-// unauthenticated request to dataset api.
-func (h *PopulationTypes) GetAreaTypes(w http.ResponseWriter, r *http.Request) {
+// NB: Logic split to private v public because
```

Comment:
Comments on these functions should still maintain the GoDoc standard of using the function/type etc name as the first word